uzdevums_6.py

```diff
@@ -0,0 +1,11 @@
+print("Ievadiet skaitli ar kuru velaties parbaudit vai sakitlis dalas ar 5 vai 7 :")
+n=input()
+if int(n)%7==0:
+  print("Ar 7 dalas")
+else:
+  print("Ar 7 nedalas")
+
+if int(n)%5==0:
```

Comment:
Piešķiršanas, salīdzīnāšanas un matemātisko darbību operatorus atdalām ar atstarpēm:
```suggestion
if int(n) % 5 == 0:
```